packages/website/components/MdxRenderer.tsx

```diff
@@ -16,66 +15,38 @@ const {
   Paragraph,
   TextLink,
   List,
-  Box,
   Table,
 } = f36Components;
 
-const styles = {
-  increesedSizeElement: css({
-    fontFamily: tokens.fontStackPrimary,
-    fontSize: `${tokens.fontSizeL}`,
-    lineHeight: `${tokens.lineHeightL}`,
-  }),
-};
-
 /* eslint-disable react/display-name */
 /* eslint-disable @next/next/no-img-element */
 const components = {
   h1: (props) => <DisplayText as="h1" {...props} />,
-  h2: (props) => (
-    <Heading
-      as="h2"
-      marginTop="spacing2Xl"
-      {...props}
-      className={styles.increesedSizeElement}
-    />
-  ),
-  h3: (props) => <Subheading as="h3" marginTop="spacingL" {...props} />,
+  h2: (props) => <Heading as="h2" marginTop="spacing2Xl" {...props} />,
+  h3: (props) => <Subheading as="h3" marginTop="spacingXl" {...props} />,
   h4: (props) => <Subheading as="h4" {...props} />,
   h5: (props) => <Subheading as="h5" {...props} />,
   h6: (props) => <Subheading as="h6" {...props} />,
-  p: (props) => (
-    <Paragraph {...props} className={styles.increesedSizeElement} />
-  ),
+  p: (props) => <Paragraph {...props} />,
```

Comment:
Didn't we want to have an increased size for all text elements?

Reply:
No
we decided to go back with the typography system that we have and, in the future, we upgrade/fix the whole system if needed